lib/how_is/pulse.rb

```diff
@@ -23,7 +23,7 @@ def text_summary
     def html_summary
       parts =
         @pulse_page_response.body
-          .split('<div class="section diffstat-summary">')
+                            .split('<div class="section diffstat-summary">')
```

Comment:
you did this in both PRs. would you mind reverting this specific line change in this PR so it doesn't conflict with the other one?